pkg/charts/dependencies.go

```diff
@@ -266,7 +265,7 @@ func UpdateHelmMetadataWithDependencies(fs billy.Filesystem, mainHelmChartPath s
 			d := &helmChart.Dependency{
 				Name:       dependencyName,
 				Condition:  fmt.Sprintf("%s.enabled", dependencyName),
-				Version:    "", // Local chart archives don't need a version
+				Version:    "",
```

Comment:
Is this still needed, or should it be removed in both places?

Reply:
Currently, this is a flow where there is a dependency in the `generated-changes/dependencies` but it is missing in the `Chart.yaml` file. In this case the Version does not exist but I will try to add some additional logic where it will be fetched from the `packages/packageName/Chart.yaml` file

Reply:
Currently, there is a logic in the `Preperation` step that intentionally skips the local charts and that's why versions are not fetched from there. This seems to be wrong in the concepts of Helm as the issue above points.
I will try to add more detailed logic that handles the local packages properly and will update the PR because this change now is just a workaround

Reply:
The new update should address the issue